docs/guides/local-apt-repository.rst

```diff
@@ -95,7 +96,7 @@ the server to check its fully qualified domain name.
 
 ``debops -l youraptserver -t apt``
 
-Make your hosts aware
+Making your hosts aware
 ---------------------
```

Comment:
The underline needs to match the header, otherwise it won't be parsed correctly by Sphinx.